README.md

```diff
@@ -0,0 +1,34 @@
+**Hosted version:** https://be-nc-news-f47c.onrender.com
```

Comment:
Is worth taking them straight to the /api endpoint so they can immediately see what available endpoints